examples/safe-math-lib/1.0.0.json

```diff
@@ -25,58 +25,60 @@
             {"name": "","type": "uint256"}
           ],
           "payable": false,
+          "stateMutability": "pure",
           "type": "function"
         },
         {
-          "constant": false,
+          "constant": true,
           "inputs": [
             {"name": "a","type": "uint256"},
             {"name": "b","type": "uint256"}
           ],
           "name": "safeAdd",
           "outputs": [
-            {"name": "","type": "uint256"}
+            {"name": "c","type": "uint256"}
           ],
           "payable": false,
+          "stateMutability": "pure",
           "type": "function"
         }
       ],
       "compiler": {
         "name": "solc",
-        "version": "0.4.6+commit.2dabbdf0.Darwin.appleclang",
+        "version": "0.4.24+commit.e67f0147.Emscripten.clang",
         "settings": {
-            "optimize": true
+          "optimize": false
         }
       },
       "natspec": {
-        "title": "Safe Math Library",
         "author": "Piper Merriam <pipermerriam@gmail.com>",
         "methods": {
           "safeAdd(uint256,uint256)": {
-            "details": "Subtracts b from a, throwing an error if the operation would cause an underflow.",
-            "params": {
-              "a": "The number to be subtracted from",
-              "b": "The amount that should be subtracted"
-            }
-          },
-          "safeSub(uint256,uint256)": {
             "details": "Adds a and b, throwing an error if the operation would cause an overflow.",
             "params": {
               "a": "The first number to add",
               "b": "The second number to add"
+            }
+		  },
```

Comment:
Nitpick.  I think there are mixed tabs and spaces here that are throwing off the indentation.

Reply:
Otherwise, I think this PR looks good.

Reply:
Good catch - do you know any handy vim tricks/plugins to deal with spaces/tabs? My vim always seems to cause a ruckus with them

Reply:
I think you just need to make sure you have your `expandtab` option set:
```
:set expandtab
```
You could also make a `json.vim` file in your `ftplugin` directory that looks something like this:
```
set shiftwidth=2
set softtabstop=2
set tabstop=2
set expandtab
```